client/src/components/InviteModal.js

```diff
@@ -0,0 +1,85 @@
+import {
+  Typography,
+  Button,
+  Modal,
+  Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc.js';
+import { postNewGroupMember } from '../lib/api';
+
+
+dayjs.extend(utc);
```

Comment:
I'm curious if you have to do this every time you use `dayjs` or if you can just do it once, like in your index.js.